### ISSUE TITLE
Integration of PinChange INT on sleep

### ISSUE THREAD

**Issue by RobertMySens** [mysensors/MySensors#1206]
Is it possible to add the possibilities to wake up from a sleep 
by a PinChange interrupt ?

Thanks for your help

**Comment by mfalkvidd** [comment]
Yes. See 
https://www.mysensors.org/download/sensor_api_20#sleeping

**Comment by RobertMySens** [comment]
Yes, totally agree with you, but whit this you just have access to INT pin of the CPU (2 in the nano case) and not other IO that you can also configure as Interrupt wake up.
